code/game/objects/items/devices/radio/radio.dm

```diff
@@ -210,6 +211,9 @@
 		return
 	if(!M.IsVocal())
 		return
+	if(radio_sounds.len) //Sephora - Radios make small static sounds now.
+		var/sound/radio_sound = pick(radio_sounds)
+		playsound(M.loc, radio_sound, 50, 1)
```

Comment:
I think this should be played to the person sending the message but not the people around them

Reply:
Internal syndicate radio go _brrr_

Reply:
Internal radios should have the sound list nulled.

Reply:
Internal radios are now silent